### DISCUSSION TITLE
Followed instruction to install locally - can't make one functional Run

### DISCUSSION THREAD

**Discussion by Carou02** [clientIO/joint#2556] [Q&A]
### Introduction

Hi, I can tell I have a good understanding on how HTML/CSS work, with basic knowledge of Javascript. Still, even if I followed the recommendations and installed the dependencies on my system to work locally with JointJS. I'm facing the same bug and can't do one single Run with the basic code provided from the tutorial. 

![Capture00](https://github.com/clientIO/joint/assets/87671971/502a5041-651e-4714-a1d6-2db14f7065c3)

### Steps to reproduce

![Capture00](https://github.com/clientIO/joint/assets/87671971/145b2f2a-5712-4434-bab0-fa6d8c2b3cbc)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by kumilingus** [comment]
Please try to use version `v4.0.1` instead.

**Comment by jamesgeorgewilliams** [comment]
Please also use the updated CDN service in this tutorial instead.
https://resources.jointjs.com/tutorial/installation

https://www.jsdelivr.com/package/npm/@joint/core
https://www.jsdelivr.com/package/npm/@joint/layout-directed-graph
